services/reference/optimism/json-rpc-methods/index.md

```diff
@@ -6,15 +6,15 @@ description: Supported standard Ethereum methods on Optimism network.
 # JSON-RPC methods
 
 The standard Ethereum methods documented here are supported by Infura on the Optimism network. For custom Optimism
-methods, please see [Optimism Ethereum JSON-RPC API documentation](https://community.optimism.io/docs/developers/build/json-rpc/) (Bedrock release).
+methods, see [Optimism Ethereum JSON-RPC API documentation](https://docs.optimism.io/builders/node-operators/json-rpc) (Bedrock release).
 
 :::info
 
-The Optimism Bedrock migration supports different JSON-RPC methods. Refer to the [Optimism Bedrock documentation](https://community.optimism.io/docs/developers/bedrock/#json-rpc) for more information.
+The Optimism Bedrock migration supports different JSON-RPC methods. Refer to the [Optimism Bedrock documentation](https://docs.optimism.io/stack/getting-started#the-op-stack-today) for more information.
```

Comment:
```suggestion
The Optimism Bedrock migration supports different JSON-RPC methods. Refer to the [Optimism Bedrock documentation](https://docs.optimism.io/builders/node-operators/json-rpc) for more information.
```
Perhaps just link to the API docs here?

Reply:
I am not sure the statement makes sense if we link to the same doc. What happened with the Bedrock migration? Are the Optimism Ethereum and Bedrock  JSON-RPC now the same?

Reply:
The Bedrock migration happened a while ago. I think even Optimism has removed references to Bedrock from it's pages. So perhaps just remove the adminition.